### ISSUE TITLE
Connecting to DAITA without a connection encryption padlock

### ISSUE THREAD

**Issue by kubatron** [mullvad/mullvadvpn-app#7299]
### Is it a bug?

- [X] I know this is an issue with the app, and contacting Mullvad support is not relevant.

### I have checked if others have reported this already

- [X] I have checked the issue tracker to see if others have reported similar issues.

### Current Behavior

When using the application on Fedora 41 KDE (Linux) with the following options enabled:
- DAITA
- Quantum Computer Attack Resistance
- Lockdown Mode
- Multi-hops
- DNS Blocking Features
- Local Area Network Sharing

The icon tray is missing the connection security padlock (connection encryption) next to the signal quality icon.

Desktop app is installed via official RPM package from Mullvad.

### Expected Behavior

When only the DAITA option is disabled, a padlock appears in the icon tray informing about the connection being secured (encrypted).

I don't know if this is only a bug related to icon or other, but I would like to see a encryption connection.

When is enabled DAITA the connection is SECURE but the ICON without lock is little bit weird.

### Steps to Reproduce

1. Open **Mullvad-VPN**
2. Enable all option from my description including **DAITA**
3. Connection in Mullvad-VPN app is OK
4. Tray icon in wireless is showing without lock
5. All connection are secure, this is check by official Mullvad website
6. Disabling **ONLY** option **DAITA** and the icon on icon tray is showing lock

### Failure Logs

```shell
.
```

### Operating system version

Fedora 41 KDE ( Kernel version: 6.11.10-300.fc41.x86_64 )

### Mullvad VPN app version

2024.8

### Additional Information

**DAITA:** enabled
![Zrzut ekranu_20241209_085116](https://github.com/user-attachments/assets/402d56da-95b4-46e1-aab9-065eb0ab07e0)

**DAITA:** disabled
![Zrzut ekranu_20241209_085136](https://github.com/user-attachments/assets/74b7acf5-2cc6-486e-8dd3-6b488b3db775)

**Comment by hulthe** [comment]
TL;DR: Your connection is encrypted, even if the padlock isn't showing.
The absence of the padlock is just a quirk of how the it works, and how we set up the VPN.

### Technical stuff below:
By default, mullvad uses the [Wireguard](https://www.wireguard.com/) implementation that's built into Linux. The network applet is able to detect this and show the padlock when you're using the VPN.

DAITA however, is not a standard Wireguard feature, and thus we don't use the standard Wireguard implementation. Instead, we ship our own implementation, which is only used when you enable DAITA. The network applet can't detect our custom Wireguard implementation, ergo it does not show the padlock.
